src/hacbs/components/IntegrationTestsListView/IntegrationTestsListView.tsx

```diff
@@ -0,0 +1,209 @@
+  const [integrationTests, integrationTestsLoaded] = useK8sWatchResource<
+    IntegrationTestScenarioKind[]
+  >({
+    groupVersionKind: IntegrationTestScenarioGroupVersionKind,
+    namespace,
+    isList: true,
+  });
+  const [nameFilter, setNameFilter] = React.useState<string>('');
+
+  const applicationIntegrationTests = React.useMemo(
+    () =>
+      integrationTestsLoaded
+        ? integrationTests?.filter((test) => test.spec.application === applicationName)
+        : [],
+    [integrationTests, applicationName, integrationTestsLoaded],
+  );
+
+  const filteredIntegrationTests = React.useMemo(
+    () =>
+      nameFilter
+        ? applicationIntegrationTests.filter(
+            (test) => test.metadata.name.indexOf(nameFilter) !== -1,
+          )
+        : applicationIntegrationTests,
+    [nameFilter, applicationIntegrationTests],
+  );
+
+  const loading = (
+    <Bullseye className="pf-u-mt-md" data-test="integration-tests__loading">
+      <Spinner />
+    </Bullseye>
+  );
+
+  if (!integrationTestsLoaded) {
+    return loading;
+  }
+
+  if (!applicationIntegrationTests?.length) {
+    return <IntegrationTestsEmptyState />;
+  }
+  const onClearFilters = () => setNameFilter('');
+  const onNameInput = (name: string) => setNameFilter(name);
+
+  return (
+    <>
+      <Title headingLevel="h4" className="pf-u-mt-lg pf-u-mb-sm">
+        Integration test pipelines
+      </Title>
+      <Flex spaceItems={{ default: 'spaceItemsXs' }}>
+        <FlexItem>
+          <TextContent>
+            <Text component={TextVariants.small}>
+              Add an integration test pipeline to test all your components after you commit code.
+            </Text>
+          </TextContent>
+        </FlexItem>
+        <FlexItem>
+          <ExternalLink
+            href="#"
+            text={
+              <Flex
+                spaceItems={{ default: 'spaceItemsXs' }}
+                justifyContent={{ default: 'justifyContentCenter' }}
+              >
+                <FlexItem className="pf-u-font-size-sm">
+                  Learn more about setting up an integration test pipeline
+                </FlexItem>
+                <FlexItem>
+                  <ExternalLinkAltIcon />
+                </FlexItem>
+              </Flex>
+            }
+          />
+        </FlexItem>
+      </Flex>
+      <>
+        <Toolbar data-testid="component-list-toolbar" clearAllFilters={onClearFilters}>
+          <ToolbarContent>
+            <ToolbarGroup alignment={{ default: 'alignLeft' }}>
+              <ToolbarItem>
+                <Button variant="control">
+                  <FilterIcon /> {'Name'}
+                </Button>
+              </ToolbarItem>
+              <ToolbarItem>
+                <TextInput
+                  name="nameInput"
+                  data-testid="name-input-filter"
+                  type="search"
+                  aria-label="name filter"
+                  placeholder="Filter by name..."
```

Comment:
on clearing the filter, the search string is not resetting.

![name-filter](https://user-images.githubusercontent.com/9964343/185384289-95e400e9-d354-44eb-8ae0-dac8835a9eed.gif)

```suggestion
                  placeholder="Filter by name..."
                  value={nameFilter}
```